stable/aws-vpc-cni/README.md

```diff
@@ -66,3 +60,18 @@ $ helm install --name aws-vpc-cni --namespace kube-system eks/aws-vpc-cni --valu
 ## Adopting the existing aws-node resources in an EKS cluster
 
 If you do not want to delete the existing aws-node resources in your cluster that run the aws-vpc-cni and then install this helm chart, you can adopt the resources into a release instead. This process is highlighted in this [PR comment](https://github.com/aws/eks-charts/issues/57#issuecomment-628403245). Once you have annotated and labeled all the resources this chart specifies, enable the `originalMatchLabels` flag on the helm release and run an update. If you have been careful this should not diff and leave all the resources unmodified and now under management of helm.
+
+Here is an example script to modify the existing resources:
+WARNING: Substitute YOUR_HELM_RELEASE_NAME_HERE with the name of your helm release.
+```
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+for kind in ds clusterRole clusterRoleBinding serviceAccount; do
```

Comment:
I think we need to annotate the `eniconfigs.crd.k8s.amazonaws.com` CRD too.

Although it's not a great idea, someone using `crd.create=true` (which is the default) will need it annotated to adopt it, as seen at https://github.com/aws/eks-charts/issues/57#issuecomment-587581045.

----

Alternatively, moving the CRD to [Helm v3's approach](https://helm.sh/docs/topics/charts/#custom-resource-definitions-crds) (`crds/` directory instead of `templates/`) would remove this need, as Helm would no longer _own_ the CRD, but merely install it if not already present. That requires Helm v2 users to manually install the CRD, which would need to be documented at least [until Helm v2 terminates support in November 2020](https://helm.sh/blog/covid-19-extending-helm-v2-bug-fixes/).

We can see the latter flow in the [`appmesh-controller` chart](https://github.com/aws/eks-charts/tree/master/stable/appmesh-controller) as mentioned at https://github.com/aws/eks-charts/issues/57#issuecomment-587588998

Reply:
Also, a nit: `daemonSet` instead of `ds`? The rest are the long-form, so this would be nice and consistent.

I'd also capitalise the first letter, as seen in the files themselves, even though they're case-insensitive. I may be in the minority on this though. And it's only documentation.

Reply:
I have changed the ds to daemonSet and added a comment.
I don't think this PR should move the existing resources, and I don't think that we should import them as helm cant manage the lifecycle of them anyway.

Reply:
I agree, but right now the chart does try to treat the CRD as a regular resource by default, and will clash with the existing CRD. We may have to tell users adopting resources to set `crd.create=false` so that it leaves the CRD alone.

(Unless Helm v3 has some magic to ignore CRDs in a template. I believed it only ignored them when installed by the crd-install hook used by Helm v2, but otherwise would treat them as normal resources.)

Reply:
I've added a comment. I think if I move this it will affect existing releases and would be a breaking change which isn't the aim of the pr. Its best left to another pr i feel.